src/Symfony/Bundle/SecurityBundle/Resources/config/security.php

```diff
@@ -102,6 +103,12 @@
             ])
             ->tag('controller.argument_value_resolver', ['priority' => 120, 'name' => UserValueResolver::class])
 
+        ->set('security.security_token_value_resolver', SecurityTokenValueResolver::class)
+            ->args([
+                service('security.token_storage'),
+            ])
+            ->tag('controller.argument_value_resolver', ['priority' => 120])
```

Comment:
this is missing the name attribute

Reply:
such Argument Resolver has no any attribute. We are injecting any sub-class of `TokenInterface` interface. Behavior is simpler comparing to `#[CurrentUser]`

Reply:
See 6 lines above, we still give the corresponding resolver a name.
This allows short-circuiting the resolver chain by using `#[MapResolver(UserValueResolver::class)]`

Reply:
@nicolas-grekas done